sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -199,10 +207,52 @@ case class InSubquery(values: Seq[Expression], query: ListQuery)
   override def children: Seq[Expression] = values :+ query
   override def nullable: Boolean = children.exists(_.nullable)
   override def foldable: Boolean = children.forall(_.foldable)
-  override def toString: String = s"$value IN ($query)"
-  override def sql: String = s"(${value.sql} IN (${query.sql}))"
+
+  private def toString(left: String, right: String): String = {
+    s"$left $symbol ($right)"
+  }
+
+  override def toString: String = toString(value.toString, query.toString)
+  override def sql: String = toString(value.sql, query.sql)
+}
+
+object PredicateSubquery {
+
+  def unapply(p: PredicateSubquery)
+    : Option[(Seq[Expression], ListQuery, (Expression, Expression) => Expression)] = {
```

Comment:
@francis0407 Is the the right format ? Not sure if this should be 2 or 4 spaces aligned.

Reply:
Databricks Scala guide gives an example:
``` scala
def newAPIHadoopFile[K, V, F <: NewInputFormat[K, V]](
    path: String,
    fClass: Class[F],
    kClass: Class[K],
    vClass: Class[V],
    conf: Configuration = hadoopConfiguration)
  : RDD[(K, V)] = {
  // method body
}
```
Therefore this should start a new line with 2 spaces indent.

Reply:
@francis0407 OK.